dist/index.html

```diff
@@ -2,12 +2,18 @@
 <html ng-app="blocChat">
```

Comment:
You can add the `dist` directory to your gitignore. It's the "compiled" version of your site if you will. It doesn't need to be versioned.